src/main/java/org/simulator/io/CSVImporter.java

```diff
@@ -140,6 +138,10 @@ public MultiTable readMultiTableFromCSV(Model model, String pathname) throws IOE
     return null;
   }
 
+  private String[] splitString(String s, String separator) {
+    return s.split(separator);
+  }
+
```

Comment:
Unclear why this method is needed.

Reply:
Sorry for this, I added it by mistake. Thanks for showing it.

Reply:
Removed in the latest commit!